Cmake Compilation Failure on Linux

Hello,

I'm trying to compile the code from source using make and I got the following error:
```
Building CXX object src/tests/interop/CMakeFiles/interop_gtests.dir/logic/image_table_logic_test.cpp.o

In file included from /illumina/scratch/iPEG/aaminfar/install/interop/interop-1.0.8/src/tests/interop/logic/image_table_logic_test.cpp:8:
/illumina/scratch/iPEG/aaminfar/install/interop/build/external/gtest/src/gtest/include/gtest/gtest.h: In function ‘testing::AssertionResult testing::internal::CmpHelperEQ(const char*, const char*, const T1&, const T2&) [with T1 = long unsigned int, T2 = illumina::interop::model::table::column_type]’:
/illumina/scratch/iPEG/aaminfar/install/interop/build/external/gtest/src/gtest/include/gtest/gtest.h:1485:   instantiated from ‘static testing::AssertionResult testing::internal::EqHelper<lhs_is_null_literal>::Compare(const char*, const char*, const T1&, const T2&) [with T1 = size_t, T2 = illumina::interop::model::table::column_type, bool lhs_is_null_literal = false]’
/illumina/scratch/iPEG/aaminfar/install/interop/interop-1.0.8/src/tests/interop/logic/image_table_logic_test.cpp:132:   instantiated from here
/illumina/scratch/iPEG/aaminfar/install/interop/build/external/gtest/src/gtest/include/gtest/gtest.h:1448: error: comparison between signed and unsigned integer expressions
gmake[2]: *** [src/tests/interop/CMakeFiles/interop_gtests.dir/logic/image_table_logic_test.cpp.o] Error 1
gmake[1]: *** [src/tests/interop/CMakeFiles/interop_gtests.dir/all
```
My linux version: CentOS release 6.7 (Final)
GCC version : gcc version 5.2.0 (GCC)

--- a/src/tests/interop/logic/image_table_logic_test.cpp
+++ b/src/tests/interop/logic/image_table_logic_test.cpp
@@ -129,7 +129,7 @@ TEST(imaging_table, test_row0)
     std::vector< bool > filled_columns;
     logic::table::populate_imaging_table(metrics, table, filled_columns);
     ASSERT_EQ(table.size(), 3u);
-    ASSERT_EQ(filled_columns.size(), model::table::ImagingColumnCount);
+    ASSERT_EQ(filled_columns.size(), static_cast<size_t>(model::table::ImagingColumnCount));
     EXPECT_TRUE(filled_columns[0]);
     EXPECT_TRUE(filled_columns[1]);
     EXPECT_TRUE(filled_columns[model::table::ErrorRateColumn]);

